client/src/main/java/de/hellfish/ngrok/client/ClientRunner.java

```diff
@@ -0,0 +1,71 @@
+package de.hellfish.ngrok.client;
+
+import de.hellfish.ngrok.utils.HttpRequestSerializer;
+import de.hellfish.ngrok.utils.MyHttpRequest;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.boot.CommandLineRunner;
+import org.springframework.stereotype.Component;
+
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.io.InputStreamReader;
+import java.io.PrintWriter;
+import java.net.Socket;
+
```

Comment:
Remove empty line

Reply:
Between class annotations and import could be 2 empty lines as far as i understand